internal/common/error_utils.go

```diff
@@ -10,9 +11,14 @@ import (
 	"github.com/go-openapi/swag"
 	"github.com/jinzhu/gorm"
 	"github.com/openshift/assisted-service/models"
-	"github.com/openshift/assisted-service/pkg/s3wrapper"
 )
 
+type NotFound string
```

Comment:
if it's already a common error why not to use `NewApiError(http.StatusNotFound, err)`?

Reply:
I did not change the existing code. Just moved it from one package to another to handle the cyclic dependency problem. This could be addressed in a different PR